### PR TITLE
1104 Add fetch remote step

### PR DESCRIPTION
**Pull request by daveatweaviate**
### What's being changed:

Adds a missing git config step 

[issue](https://github.com/weaviate/weaviate-io/pull/1104)

### Type of change:

- [x] **Documentation** updates (non-breaking change to fix/update documentation)

### How Has This Been Tested?

- [ ] **Github action** – automated build completed without errors
- [ ] **Local build** - the site works as expected when running `yarn start`

### CODE REVIEW

**Comment by daveatweaviate** [comment]
Would you have a look at this update when you have a moment? Thanks!

File: README.md
Comment (databyjp) [inline README.md:1]: Hey @daveatweaviate 👋

This is nice to have.

I don't know if I want to be very prescriptive about how to use git, as everybody has different workflows with GitHub desktop / VSCode / PyCharm extensions and so on. Even for me, for example, I prefer to use git switch over git checkout when it comes to branches. (I think switch looks for remote branches and sets up tracking if it doesn't exist)

How about we rephrase these sections as a possible suggested workflow?

Comment (daveatweaviate) [reply README.md:1]: Thanks for the suggestion. I think everything above line 89 is required (at least if you clone using ssh). 

I added some text around the checkout step to show there might be other ways of doing that part. Let me know if I should move the 'there are alternate ways' step up a bit.

**Review by daveatweaviate — COMMENTED** [review]
@databyjp thanks for the feedback. This is ready for another look

**Pull request merged**